Login.html

```diff
@@ -0,0 +1,31 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <meta http-equiv="X-UA-Compatible" content="ie=edge">
+    <title>Trainwise - Login page</title>
+
+</head>
+
+<body>
+
+    <h1>Trainwise</h1>
+
+    <h2>Login to your account</h2>
+    <form>
+        <input type="text" title="username" placeholder="username" />
+        <input type="password" title="password" placeholder="password" />
```

Comment:
Input fields need a `label` element with a `for` attribute to tell the screen readers about the input functionality. `placeholder` attributes are only for example content and can't replace labels.

Reply:
Thanks